api/envoy/config/metrics/v3/stats.proto

```diff
@@ -259,6 +262,46 @@ message TagSpecifier {
   }
 }
 
+// Specifies a matcher for stats and the buckets that matching stats should use.
+message HistogramBucketSettings {
+  // The stats that this rule applies to.
+  type.matcher.v3.StringMatcher match = 1 [(validate.rules).message = {required: true}];
+
+  // Each value is the upper bound of a successive bucket.
+  repeated double buckets = 2 [(validate.rules).repeated = {min_items: 2}];
+}
+
+// Settings for how histograms are emitted.
+message HistogramSettings {
```

Comment:
The reason for this extra message with only 1 field is so that someone can easily add configurable quantiles later if they want. It would be the 2nd field in this message. Not sure if the extra message is worth it or not though.

Reply:
Would you want the quantiles to be global or per histogram though? In the per histogram case wouldn't it go above negating the need for this message?

Reply:
I honestly haven't put any thought into quantiles. It's more likely that global quantile config would be useful than global histogram buckets.

Should I delete this intermediate message, and if/when someone does quantile settings they can add the appropriate bits to the parent message?

Reply:
That would be my vote but I don't feel strongly about it.

Reply:
@mattklein123 fixed. Config look good now?